### PR TITLE
#162782158 Created user serializers

### PR DESCRIPTION
**Pull request by jamesbeamie**
**What does this PR do?**
- Merges into develop user serializers 
**Description of Task to be completed?**
- Import user models.
- Create a user serializer class.
- Create a method for user validation.
 **How should this be manually tested?**
- Git clone and paste the url on the terminal
- Run python3 manage.py runserver
- Open the url in the browser and fill in the fields
**What are the relevant pivotal tracker stories?**
- https://www.pivotaltracker.com/story/show/162782158

### CODE REVIEW

**Review by tesh254 — APPROVED** [review]
Nice use of default Django models, but leave an extra line for every file you are done with, it is a PEP8 standard

**Comment by Ruiru11** [comment]
Good approach just correct on what @tesh254  has pointed out

**Comment by danuluma** [comment]
Nice work done. Please remember to adhere to pep8 in the next pr

**Review by danuluma — APPROVED** [review]
Good work

**Review by Ruiru11 — APPROVED** [review]

File: todo/todoapp/serializers.py [hunk 1]
```diff
@@ -13,4 +14,27 @@ class Meta:
         """To serialize fields with the model fields."""
         model = TodoList
         fields = ('id', 'name', 'description', 'done', 'date_created')
```
Comment (tesh254) [inline todo/todoapp/serializers.py:16]: Leave an extra line it is a PEP8 standard

**Pull request merged**